styles/GlobalStyle.js

```diff
@@ -60,24 +45,29 @@ const GlobalStyle = createGlobalStyle`
   button {  
     font-size: 1rem;
     margin: 1rem;
+    background-color: transparent;
+    border-radius: 0.5rem;
+    padding: 0.5rem
   }
   
   article {
     display: flex;
     flex-direction: column;
-    background-color: orange;
-    border-radius: 0.5rem 0.5rem 0 0;
+    border-radius: 0.75rem ;
     box-shadow: 0 0 1rem 0 #000;
+    background-color: rgb(255, 251, 224);
+    width: 100vw;
```

Comment:
this can easily lead to unintended behaviour like elements reaching out of the normal viewport. If the article is inserted into a container with padding, the total width of the container will be 100vw plus the padding. This will result in a webpage that is wider than the window, you will get horizontal scrollbars etc. You can use 100% here or just omit it, since articles should have 100% width by default.

Reply:
Yes, it actually does reach out sometimes 😅
Removed it and will search for a better solution.